website/content/docs/secrets/pki/est.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 layout: docs
-page_title: Enrollment over Secure Transport (EST) within Vault | PKI - Secrets Engines
-description: An overview of the Enrollment over Secure Transport protocol implementation within Vault.
+page_title: Enrollment over Secure Transport (EST)
+description: >-
+   Understand the configuration and limitations of Vault's PKI secrets engine implementation of the EST protocol.
```

Comment:
Please consider spelling out "Enrollment over Secure Transport" in the description, to match the descriptions of the CIEPS and CMPv2 pages.